devel/distcc/files/patch-no-faccessat.diff

```diff
@@ -0,0 +1,36 @@
+--- src/orig.serve.c	2018-03-08 00:01:17.000000000 +0100
++++ src/serve.c	2018-05-17 18:45:47.000000000 +0200
+@@ -370,25 +370,29 @@
+  **/
+ static int dcc_check_compiler_whitelist(char *compiler_name)
+ {
++#ifndef AT_SYMLINK_NOFOLLOW
```

Comment:
It does not look like this macro is used anywhere in this function. Why do you need to check for this here? Is the intention really to disable the whitelist completely and allow execution of arbitrary commands?